src/modules/LoggedIn/Project/ProjectFilter.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import ProjectForm from "./ProjectForm";
+import { Row, Input, Select, Form } from "antd";
+const { Option } = Select;
+const { Search } = Input;
+const ProjectFilter = () => {
+  return (
+    <Row>
+      <Form>
+        <label style={{ marginRight: 10 }}>Sort By</label>
+        <Select defaultValue="newest" style={{ width: 120, marginRight: 10 }}>
+          <Option value="newest">Newest</Option>
+          <Option value="oldest">Oldest</Option>
+          <Option value="name">Name</Option>
+        </Select>
```

Comment:
terus kalau bikin label, pastikan dikasih attribute for, terus masukin id dari input elementnya, jadi ketika text labelnya di klik, kamu bakal auto focus ke inputannya. Nanti bisa mempermudah pas bikin testingnya

coba lihat ini buat contoh cara pake attribute for di label
https://www.w3schools.com/tags/att_label_for.asp

Reply:
Atau kalau pake ant design, bisa pake component Form.Item 
https://ant.design/components/form/#Form.Item

Reply:
kalo pake Form.Item apakah ditambahkan id sama input juga mas ato cukup pake name aja?

Reply:
harusnya cukup kyk gini sih. `colon={false}` itu supaya labelnya gak ditambahi `:` sama antd

```javascript
<Form.Item
        label="Username"
        name="username"
        colon={false}
      >
        <Input />
      </Form.Item>
```

Reply:
kalo pake Form.Item itu style nya ngaruh mas jadi letaknya agak ke atas, ato di manupulasi pake style biasah aja mas?

Reply:
component Row nya coba dikasih align items, pake props  align="middle"

Reply:
itu rownya udah saya kasih middle mas tapi gak ngaruh juga

Reply:
coba kamu push dulu, ntar aku cek di preview app nya

Reply:
siap sudah mas

Reply:
By default `Form.Item` ini punya default margin bottom 24px. Maka dari itu dia bakal gak lurus walaupun diterapin `align="middle"`, karena `Form.Item` nya bakal di push ke atas sebanyak 24px. Maka dari itu ketika kamu tambahin margin top, dia menjadi lurus, karena setelah di push keatas sebanyak 24px, kamu push kebawah lagi sebanyak 24px, sehingga jadi netral

Reply:
solusi simplenya, margin nya dibuat 0 aja `style={{ margin: 0 }}`. Harusnya aman

Reply:
mantab udah solved mas terima kasih.